platform/security/platform-security-core-api/src/test/java/ddf/security/samlp/SimpleSignTest.java

```diff
@@ -100,7 +101,9 @@ public void setUp() throws Exception {
         pemWriter.flush();
         dsaCert = writer.toString()
                 .replace("-----BEGIN CERTIFICATE-----", "")
-                .replace("-----END CERTIFICATE-----", "");
+                .replace("-----END CERTIFICATE-----", "")
+                .replace("\n", "");
```

Comment:
Is this standards compliant? What if the newlines aren't stripped from an actual certificate?